### PR TITLE
bump es version, remove default storageClassName

### PR DESCRIPTION
**Pull request by kimxogus**
This pr
1. Bump default es version to `6.7.1`
2. Removes `standard` storageClassName by default as this value will be default storage class if empty(you don't need to specify this). In aws environment, default storage class is generally `gp2`, so `volumeClaimTemplate` should be always overridden.

### CODE REVIEW

**Comment by kimxogus** [comment]
should be `7.0.0`?

**Comment by z0lope0z** [comment]
👍 trying to helm install on new AWS EKS cluster and getting the error

persistentvolume-controller  storageclass.storage.k8s.io "standard" not found

File: elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ maintainers:
 - email: helm-charts@elastic.co
   name: Elastic
 name: elasticsearch
-version: 6.6.2-alpha1
```
Comment (Crazybus) [inline elasticsearch/Chart.yaml]: Can you remove this version bump please. All versions of the charts are bumped and released at the same time (just like with all of the elastic stack). I went into more details in this PR if you are interested in the reasons why: https://github.com/elastic/helm-charts/pull/40#discussion_r249422024

File: elasticsearch/values.yaml [hunk 2]
```diff
@@ -71,7 +71,6 @@ networkHost: "0.0.0.0"
 
 volumeClaimTemplate:
   accessModes: [ "ReadWriteOnce" ]
-  storageClassName: "standard"
```
Comment (Crazybus) [inline elasticsearch/values.yaml]: Awesome! This is much better. Can you also update the defaults in the readme to match https://github.com/elastic/helm-charts/tree/master/elasticsearch#configuration

**Comment by Crazybus** [comment]
jenkins test this please

File: elasticsearch/values.yaml (hunk 2 above)
Comment (kimxogus) [reply elasticsearch/values.yaml]: updated readme

File: elasticsearch/Chart.yaml (hunk 1 above)
Comment (kimxogus) [reply elasticsearch/Chart.yaml]: reverted chart version bump

File: elasticsearch/Chart.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ maintainers:
   name: Elastic
 name: elasticsearch
 version: 6.6.2-alpha1
-appVersion: 6.6.2
+appVersion: 6.7.1
```
Comment (Crazybus) [inline elasticsearch/Chart.yaml]: Something went wrong with the revert as the version bump is still here.

File: elasticsearch/README.md [hunk 4]
```diff
@@ -55,14 +55,14 @@ If you currently have a cluster deployed with the [helm/charts stable](https://g
...
-| `imageTag`                 | The Elasticsearch docker image tag                                                                                                                                                                                                                                                                                         | `6.6.2`                                                                                                                   |
+| `imageTag`                 | The Elasticsearch docker image tag                                                                                                                                                                                                                                                                                         | `6.7.1`                                                                                                                   |
```
Comment (Crazybus) [inline elasticsearch/README.md]: This one should be reverted too.

File: elasticsearch/values.yaml [hunk 5]
```diff
@@ -43,7 +43,7 @@ secretMounts: []
 #    path: /usr/share/elasticsearch/config/certs
 
 image: "docker.elastic.co/elasticsearch/elasticsearch"
-imageTag: "6.6.2"
+imageTag: "6.7.1"
```
Comment (Crazybus) [inline elasticsearch/values.yaml]: This one too

**Pull request merged**